bihstaging.data-commons.org/portal/gitops.json

```diff
@@ -111,14 +111,14 @@
   "requiredCerts": [],
   "featureFlags": {
     "explorer": false,
-    "discovery": true,
-    "discoveryUseAggMDS": true,
-    "discoveryUseAggWTS": true,
-    "workspaceRegistration": true,
-    "workspaceTokenServiceRefreshTokenAtLogin": true
+    "discovery": false,
```

Comment:
I think we should also hide these buttons and nav links for discovery page. I remember thanh was working on something similar (here -- [gitops-qa/pull/2962](https://github.com/uc-cdis/gitops-qa/pull/2962/files))

Reply:
yes we can, but we don't use the portal since we use FEF, so the buttons should not be visible. Ideally we'll delete the portal and its config eventually

we can still do some cleanup but it's not needed for this PR which is just to unblock submission progress